### DISCUSSION TITLE
UI beim Bearbeiten von Objekten

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#142] [General]
Beim Bearbeiten bestehender Objekte im webGIS wird die Layerschaltung abgedeckt.
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/608a28bc-50a9-42c1-b9ea-eff1f634dd06)
Es ist zwar praktikabel, dass man die Attributfelder vergrößern kann, aber wenn man den Hintergrund ändern oder zusätzliche Themen einschalte will, geht das nicht mehr. (Build 5.23.5101)

Wünschenswert wäre es, wie bei "Neues Objekt anlegen", dass der Zugriff auf die Layerschaltung möglich ist.
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/7ede94bf-85f5-40f1-a8db-3283203fb95c)

**Comment by gis-eni** [comment]
Das Editing wird/wurde für den Desktop Anwender mit **WebGIS 6** überarbeitet. Diese Verschlag ist dort auch schon umgesetzt:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/4d53701a-fc81-45d4-8752-9e2ff4e5abc3)

**Reply by marko-jer** [reply]
Das heißt in webGIS kommt diese Änderung nicht mehr ... warten auf 6?

**Reply by gis-eni** [reply]
Genau. Ihr könnt aber schon anfangen WebGIS 6 zu verwenden

**Reply by marko-jer** [reply]
Gibt es dann bald auch eine neue Release? in der zur Verfügung stehenden steht noch "(alpha - not for production)"

**Reply by gis-eni** [reply]
Werde bis nächsten Mittwoch wieder einmal eine (Beta) Release auf den Server stellen.
Die Finale Version kommt dann beim Workshop

**Reply by marko-jer** [reply]
OK! Danke!
